app/jobs/gpo_expiration_job.rb

```diff
@@ -0,0 +1,60 @@
+class GpoExpirationJob < ApplicationJob
+  queue_as :low
+
+  def initialize(analytics: nil)
+    @analytics = analytics
+  end
+
+  def perform(as_of: Time.zone.now, limit: nil)
```

Comment:
I feel like using the name `now` might be clearer?
```suggestion
  def perform(now: Time.zone.now, limit: nil)
```